jetstream/jetstream.go

```diff
@@ -420,6 +422,24 @@ func (js *jetStream) UpdateStream(ctx context.Context, cfg StreamConfig) (Stream
 	}, nil
 }
 
+func (js *jetStream) CreateOrUpdateStream(ctx context.Context, cfg StreamConfig) (Stream, error) {
+	s, err := js.UpdateStream(ctx, cfg)
+	if err != nil {
+		if err == ErrStreamNotFound {
```

Comment:
I would suggest using `errors.Is()` instead of `==` for comparing errors for consistency with the rest of the package as well as safeguarding from error wrapping.

Reply:
I have changed it to `errors.Is()`